plugin/cache/cache.go

```diff
@@ -112,7 +112,7 @@ type ResponseWriter struct {
 // newPrefetchResponseWriter returns a Cache ResponseWriter to be used in
 // prefetch requests. It ensures RemoteAddr() can be called even after the
 // original connection has already been closed.
-func newPrefetchResponseWriter(server string, state request.Request, c *Cache) *ResponseWriter {
+func newPrefetchResponseWriter(server string, state request.Request, c *Cache, do bool) *ResponseWriter {
```

Comment:
why does this need a `do` bool? We always want to do a DO enabled lookup and fix the reply for clients that didn't ask for it.

Reply:
i made this because refresh when stale did this, it keeps the original do and not the modified

Reply:
Ok, removing this